lib/repl.js

```diff
@@ -332,18 +337,21 @@ function REPLServer(prompt,
           }
         }
       } catch (e) {
-        err = e;
-
-        if (err && err.code === 'ERR_SCRIPT_EXECUTION_INTERRUPTED') {
+        if (isExitCmd) {
+          self.outputStream.write('(To exit, press ^D or type .exit)\n');
+          return self.displayPrompt();
+        } else {
```

Comment:
I think you don’t really need to add the `else { … }` here since you `return` early in the `if() {}` block anyway.